src/client/layouts/Main.tsx

```diff
@@ -205,6 +212,11 @@ export const MainLayout = ({
 					{pageHeader && (
 						<header css={headerStyles(hasSummary)}>
 							<h1 css={[pageTitleStyles]}>{pageHeader}</h1>
+							{pageSubText && (
```

Comment:
Have we tested how the sub-text looks with the page level error, is there a story for that?

Reply:
Have added in a bunch of stories to test this!